tests/org.eclipse.xtend.ide.tests/src/org/eclipse/xtend/ide/tests/macros/EclipseFileSystemTest.xtend

```diff
@@ -99,23 +97,15 @@ class EclipseFileSystemTest extends JavaIoFileSystemTest {
 
 		val file = new Path("/bar/Foo.text")
 		assertFalse(file.exists)
-		assertNotNull(file.toURI)
-
-		file.contents = "Hello Foo"
-		assertTrue(file.exists)
-		val uri = file.toURI
-		assertNotNull(uri)
-		assertTrue('' + uri, uri.toString.endsWith('/foo/bar/Foo.text'))
-
-		val javaIoFile = new File(uri)
-		assertTrue(javaIoFile.exists)
-
-		val data = newByteArrayOfSize(javaIoFile.length as int)
-		val fis = new FileInputStream(javaIoFile)
-		fis.read(data)
-		fis.close
-
-		assertEquals("Hello Foo", new String(data))
+		assertNull(file.toURI)
```

Comment:
The change seems to deliberately break support that was added with bug https://bugs.eclipse.org/bugs/show_bug.cgi?id=423787 despite concerns. Why was it necessary to support that at that time but is no longer?